### PR TITLE
Show reviews with service

### PR DESCRIPTION
**Pull request by aurelien-jacomy**
Included the show of reviews in the service page. 

- Show the link to edit and destroy only if the current user is the author of the review
- Nested the routes for reviews
- Added the correct link in the controller

image

### CODE REVIEW

**Comment by aurelien-jacomy** [comment]
Added a second commit with the destroy method, and the confirmation before delete in the view

File: app/views/services/show.html.erb [hunk 1]
```diff
@@ -2,11 +2,43 @@
 	<div class="row justify-content-center">
 		<div class="col-sm-8">
 			<h2><%= @service.name %> by <%= @service.user.first_name %></h2>
-			<%= cl_image_tag @service.photo.key, height: 200, width: 300, crop: :fill %>
+			<!-- cl_image_tag @service.photo.key, height: 200, width: 300, crop: :fill -->
```
Comment (gabantonini) [inline app/views/services/show.html.erb:5]: aqui vc pode fazer a mesma logica que tem no INDEX do Service => se tiver foto do user, mostra a dele. se nao tiver, mostra a padrão

Comment (gabantonini) [reply app/views/services/show.html.erb:5]: eu mudei no index, mas acabei nao mudando no show

File: app/views/services/show.html.erb [hunk 2]
```diff
@@ -2,11 +2,43 @@
...
 			<p><%= @service.price %></p>
 			<p><%= @service.time_to_answer %></p>
 			<p><%= @service.disponibility %></p>
 		</div>
 	</div>
+
+	<!-- Show reviews -->
+	<div class="row justify-content-center">
+		<div class="col-sm-8">
+			<h2>Confira o que nossos usuários falarm desse serviço:</h2>
```
Comment (gabantonini) [inline app/views/services/show.html.erb:16]: falou a letra "a" em falaram

File: app/views/services/show.html.erb (hunk 1 above)
Comment (aurelien-jacomy) [reply app/views/services/show.html.erb:5]: Done ! Não sabia se era problema de branch ou não ahaha

File: app/views/services/show.html.erb (hunk 2 above)
Comment (aurelien-jacomy) [reply app/views/services/show.html.erb:16]: Thanks

**Pull request merged**